### PR TITLE
repost ui

### PR DESCRIPTION
**Pull request by jessopb**
Adds () Anonymous reposted to reposts, shows it on top claim Tested a few other claim preview locations with reposts.
Can test on bob

### CODE REVIEW

File: ui/component/claimPreview/view.jsx [hunk 1]
```diff
@@ -131,9 +131,10 @@ const ClaimPreview = forwardRef<any, {}>((props: Props, ref: any) => {
       isValid = false;
     }
   }
+  const isRepost = claim && claim.repost_url;
 
-  const contentUri = hideRepostLabel && claim && claim.repost_url ? claim.canonical_url || claim.permanent_url : uri;
-  const isChannel = isValid ? parseURI(contentUri).isChannel : false;
+  const contentUri = claim && isRepost ? claim && (claim.canonical_url || claim.permanent_url) : uri;
```
Comment (seanyesmunt) [inline ui/component/claimPreview/view.jsx:136]: This second check of `claim` isn't needed

File: ui/component/claimRepostAuthor/view.jsx [hunk 2]
```diff
@@ -16,18 +16,29 @@ function ClaimRepostAuthor(props: Props) {
...
+    return (
+      <div className="claim-preview__repost-author">
+        <Icon icon={ICONS.REPOST} size={10} />
+        <span>
+          <strong>Anonymous</strong> {__('Reposted')}
+        </span>
+      </div>
+    );
+  }
+  if (!repostChannelUrl) {
```
Comment (seanyesmunt) [inline ui/component/claimRepostAuthor/view.jsx:38]: I think this would be more descriptive if it was
```
if (!repostUrl) { }
```

**Review by seanyesmunt — CHANGES_REQUESTED** [review]
small comments

**Comment by jessopb** [comment]
Hold on this

**Pull request merged**